tests/unit/grains/test_core.py

```diff
@@ -809,6 +815,8 @@ def _check_type(key, value, ip4_empty, ip6_empty):
                             _check_type(key, value, ip4_empty, ip6_empty)
 
     @skipIf(not salt.utils.platform.is_linux(), 'System is not Linux')
+    @patch.object(salt.utils, 'is_windows', MagicMock(return_value=False))
```

Comment:
This should be `salt.utils.platform` instead of `salt.utils`. I know this wasn't your code initially, and that you just moved an existing patch, but can you make this small edit? The code in `salt/grains/core.py` uses `salt.utils.platform.is_windows()`, so the test needs it too. It won't affect this test's accuracy on *NIX, but it will on Windows.

Reply:
@terminalmage fixd